scripts/prepare_sector_network.py

```diff
@@ -425,7 +425,7 @@ def update_wind_solar_costs(n, costs):
 
             logger.info(
                 "Added connection cost of {:0.0f}-{:0.0f} Eur/MW/a to {}".format(
-                    connection_cost[0].min(), connection_cost[0].max(), tech
+                    connection_cost.min(), connection_cost.max(), tech
```

Comment:
wasn't sure with that one honestly, but using `connection_cost[0]` basically left with the min and max of a single value

also it was deprecated, with the following message: `FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use ser.iloc[pos]`

Reply:
ok, I would say.